docs/changelog.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+### 17-December-2021 - 09:56 CET
+
+- [feature] Introduce 'epoch' concept to build configurations as a strategy to build packages for new compilers.
+- [feature] Allow pull-requests to build packages from different recipes for the same library.
```

Comment:
What does it mean?

Reply:
if you have two (or more) recipes for the same library in sub-directories, e.g. `zlib/1.2.11` and `zlib/1.2.8`, you should be able to do changes in both within the single PR. previously, you was forced to open one PR per sub-directory.